.circleci/config.yml

```diff
@@ -47,5 +85,13 @@ workflows:
               ignore: /.*/
             tags:
               only:
-                - /^v[0-9]+(\.[0-9]+)*$/
-
+                - /^v[0-9]+(\.[0-9]+)*.+$/
+      - publish:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only:
+                - /^v[0-9]+(\.[0-9]+)*.+$/
```

Comment:
not a regex expert. with automatic releases, I think we need to be more strict in the tags naming. for example
```                
- /^v[0-9]\d*\.[0-9]\d*\.[0-9]\d*$/
- /^v[0-9]\d*\.[0-9]\d*\.[0-9]\d*-rc[0-9]\d*$/
```
with these we force v0.0.0 and v.0.0.0-rc0 tags naming. we can merge in one expression. 

WDYT ??

Reply:
as rebased with master, this change is already done, thanks!